### PR TITLE
Fixed exclude phenotype column command

### PR DESCRIPTION
**Pull request by jennifertran**
Based on Issue #65 

## Problem
When you include the `--exclude-resistance-phenotypes` in the command line, the Predicted Phenotypes Column is still included in the summary and detailed_summary file

## Solution
Take advantage of _polymorphism_ and _overriding_ to dynamically remove the Predicted Phenotypes Column

## Implementation
Create getter methods to return the proper column names and values

## Testing
Ran through the staramr tests and refactored the unit tests to use the `AMRDetectionSummaryResistance` method rather than the `AMRDetectionSummary`

### CODE REVIEW

File: staramr/results/AMRDetectionSummary.py [hunk 1]
```diff
@@ -59,6 +59,9 @@ def _include_negatives(self, resistance_frame: DataFrame) -> DataFrame:
 
         return resistance_frame.append(negative_entries, sort=True)
 
+    def get_detailed_negative_columns(self):
```
[the remark this answers is not in the document]
Comment (jennifertran) [reply staramr/results/AMRDetectionSummary.py]: Yeah fixed that in commit 905a8ed

**Comment by jennifertran** [comment]
@apetkau The command error is coming from including the negatives method, I forgot to add an if/else case if the phenotype command is enabled.

File: staramr/results/AMRDetectionSummary.py [hunk 2]
```diff
@@ -67,10 +73,17 @@ def _include_detailed_negatives(self, resistance_frame: DataFrame, plasmid_frame
         negative_res_names_set = names_set - resfinder_names_set
 
         negative_entries = None
+        negative_columns = self._get_detailed_negative_columns()
 
         if len(negative_res_names_set) != len(names_set) or resistance_frame.empty:
-            negative_resistance_entries = pd.DataFrame([[x, 'None', 'Sensitive', '', ''] for x in negative_res_names_set],
-                                                       columns=('Isolate ID', 'Gene', 'Predicted Phenotype', 'Start', 'End')).set_index('Isolate ID')
+
+            if self._include_phenotype():
```
Comment (apetkau) [inline staramr/results/AMRDetectionSummary.py]: I would shift this code into another method like `_get_negative_resistance_entries()`.

Also, it looks like the negative **Predicted Phenotypes** are showing up as `-` for me in the `summary.tsv` file:

image

Comment (jennifertran) [reply staramr/results/AMRDetectionSummary.py]: I've fixed that in my next commit, also the Predicted Phenotype issue is coming from a line of code where it wasn't placed properly. I moved it to the appropriate place.

Comment (apetkau) [reply staramr/results/AMRDetectionSummary.py]: Awesome. Thank you.

**Comment by apetkau** [comment]
Thanks so much. I'm merging this in.

**Pull request merged**